systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -60,6 +60,12 @@ public static void waitForClientSuccess(String jobName, String namespace, int me
             () -> kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(jobName).get().getStatus().getSucceeded().equals(1));
     }
 
+    public static void waitForClientFailure(String jobName, String namespace, long timeout) {
+        LOGGER.info("Waiting for producer/consumer:{} will be in error state", jobName);
+        TestUtils.waitFor("job finished", Constants.GLOBAL_POLL_INTERVAL, timeout,
+            () -> kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(jobName).get().getStatus().getSucceeded().equals(1));
+    }
```

Comment:
Should this be a `waitForJobFailure` and sit in `JobUtils`? apart from the log message, there is nothing client specific.